docs/src/languages/ruby.md

```diff
@@ -37,3 +37,29 @@ Solargraph has formatting and diagnostics disabled by default. We can tell Zed t
 ### Configuration
 
 Solargraph reads its configuration from a file called `.solargraph.yml` in the root of your project. For more information about this file, see the [Solargraph configuration documentation](https://solargraph.org/guides/configuration).
+
+### Setting up `ruby-lsp`
+
+Zed currently doesn't install Ruby LSP automatically. To use Ruby LSP, you need to install the gem. Zed just looks for an executable called `ruby-lsp` on your `PATH`.
+
+You can install the gem manually with the following command:
+
+```shell
+gem install ruby-lsp
+```
+
+Ruby LSP uses pull-based diagnostics which Zed doesn't support yet. We can tell Zed to disable it by adding the following to your `settings.json`:
```

Comment:
@vitallium although diagnostics isn't supported, were you encountering problems/errors when this was enabled?